assets/js/pages/Users/Users.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from '@common/Button';
+import Table from '@common/Table';
+import PageHeader from '@common/PageHeader';
+import Modal from '@common/Modal';
+import Tooltip from '@common/Tooltip';
+import Banner from '@common/Banners/Banner';
+import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
+
+const USER_CREATE_ROUTE = '/users/new';
+
+function Users({
+  handleDeleteUser = () => {},
+  navigate = () => {},
+  setModalOpen = () => {},
```

Comment:
Why do we need to pass this `setModalOpen`, `setDeleteUserId`, `deleteUserId`, and `modalOpen` from outside. They look inherent and pure for this page.

Reply:
+1 The visual impact of the component, without any external side effect should be contained in the component, with the proper state, the external actions like perform the api call for delete should be injected.